.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -101,23 +108,28 @@ https://gitlab.grml.org/grml/build-release/-/pipelines/TBD/builds
     ```
     sha256sum -c SHA256SUMS-20YY.MM *.sha256
     gpg --keyid-format long --verify SHA256SUMS-20YY.MM.gpg SHA256SUMS-20YY.MM
     for x in *iso; do gpg --keyid-format long --verify $x.asc $x; done
     ```
   - [ ] update `index.*html` in ftp-master.g.o
+- [ ] move all remaining items out of project and close it
 
-# Announce
+# Announce Release
 
 - [ ] wait for mirror sync, check https://mirror.grml.org
-- [ ] merge website branch to master: [Previous](https://github.com/grml/grml.org/pull/102) -- [Current PR 🛳](https://github.com/grml/grml.org/pull/TBD)
-- [ ] merge blog branch: [Previous](https://github.com/grml/blog.grml.org/pull/11 ) -- [Current PR 🛳](https://github.com/grml/blog.grml.org/pull/TBD)
-- [ ] mailing list: [Previous](https://lists.mur.at/pipermail/grml-announce/2025-May/000062.html)
-- [ ] mastodon [Previous](https://hachyderm.io/@grmlproject/114511194074491152)
-- [ ] bluesky [Previous](https://bsky.app/profile/grmlproject.bsky.social/post/3lp77x3xla227)
-- [ ] IRC topic
-- [ ] [netboot.xyz: create issue](https://github.com/netbootxyz/netboot.xyz): [Previous issue](https://github.com/netbootxyz/netboot.xyz/issues/1623) [Previous PRs](https://github.com/netbootxyz/debian-squash/pull/3)
+- [ ] merge website update PR to master: (*Comment: i.e. PR for [Grml Release 2025.05](https://github.com/grml/grml.org/pull/102) + [Grml Release 2025.08](https://github.com/grml/grml.org/pull/117)*)
+- [ ] merge blog post PR: (*Comment: i.e. PR for [New blogpost: Grml - new stable release 2025.05 available](https://github.com/grml/blog.grml.org/pull/11) + [New blogpost: Grml - new stable release 2025.08 available](https://github.com/grml/blog.grml.org/pull/13)*)
+- [ ] send to mailing list: (*Comment: i.e. [Grml - new stable release 2025.05 available](https://lists.mur.at/hyperkitty/list/grml-announce@ml.grml.org/thread/2TA7K3B6SVA4VMSGN6TOF2NX5XP2PLPX/) + [Grml - new stable release 2025.08 available](https://lists.mur.at/hyperkitty/list/grml-announce@ml.grml.org/thread/EPGBLAYO2N7EINV7EJUPXT255O3BZBSP/)*)
+- [ ] mastodon: (*Comment: [2025.05](https://hachyderm.io/@grmlproject/114511194074491152) + [2025.08](https://hachyderm.io/@grmlproject/115038916407200568)*)
+- [ ] bluesky: (*Comment: [2025.05](https://bsky.app/profile/grmlproject.bsky.social/post/3lp77x3xla227) + [2025.08](https://bsky.app/profile/grmlproject.bsky.social/post/3lwjlbsn2ks2z)*)
+- [ ] Update IRC topic (*Comment: how?*)
+- [ ] Create [issues @ netboot.xyz](https://github.com/netbootxyz/netboot.xyz/issues/new/choose) + PRs to bump Grml version (*Comment: i.e. [netboot.xyz#1661: Update Grml to 2025.08](https://github.com/netbootxyz/netboot.xyz/issues/1661) + [merged/closed PRs](https://github.com/netbootxyz/debian-squash/pulls?q=is%3Apr+is%3Aclosed+grml)*)
+
+# Post Release Day
+
+- [ ] update [release checklist template](https://github.com/grml/grml/blob/master/.github/ISSUE_TEMPLATE/release-checklist.md) with new learnings
```

Comment:
I added the "Post Release Day" section because I think this is a task which doesn't need to be completed on release day. If you think otherwise we can move it to the "Release Day" section.

(Or if any other tasks do not need to be completed on release day, we could move it here...)